src/components/Page.js

```diff
@@ -2,18 +2,97 @@ import React from 'react';
 import { withStyles } from '@material-ui/core/styles';
 import { Grid, Paper, Container } from '@material-ui/core';
 import PropTypes from 'prop-types';
+import BlogPost from './BlogPost';
+import weekendfly from '../images/weekendfly.jpg';
+import redclay from '../images/redclay.jpg';
+import nightfamily from '../images/nightfamily.jpg';
 
 const styles = (theme) => ({
   content: {
     flexGrow: 1,
   },
   paper: {
     padding: theme.spacing(2),
-    textAlign: 'center',
+    textAlign: 'left',
     color: theme.palette.text.secondary,
   },
 });
 
+const blogList = [
```

Comment:
I think we can make this a better app structure by creating a folder called BlogPost under the components directory. Move BlogPost.js to this folder and create a new file called blogs.js. Move this blogList to blogs.js so you can just import this array and use it here. Msg me if you have any questions